frontend/public/components/masthead.tsx

```diff
@@ -9,12 +10,96 @@ import * as dedicatedLogoImg from '../imgs/openshift-dedicated-logo.svg';
 import { FLAGS, connectToFlags, flagPending } from '../features';
 import { authSvc } from '../module/auth';
 import { Dropdown, ActionsMenu } from './utils';
+import { SoftwareDetails } from './software-details';
 
 import { coFetchJSON } from '../co-fetch';
 import { SafetyFirst } from './safety-first';
 
 const developerConsoleURL = (window as any).SERVER_FLAGS.developerConsoleURL;
 
+export class HelpMenu extends React.Component<HelpMenuProps, HelpMenuState> {
+  constructor (props) {
+    super(props);
+    this.state = { showModal: false };
+    this.close = this.close.bind(this);
+    this.open = this.open.bind(this);
+    this.redirect = this.redirect.bind(this);
+  }
+
+  close() {
+    this.setState({ showModal: false });
+  }
+
+  open() {
+    this.setState({ showModal: true });
+  }
+
+  redirect() {
+    window.open('https://docs.openshift.com/online/welcome/index.html', '_blank');
```

Comment:
It would be much better to make this an actual link if we can. This has security implications because we can't set `rel=noreferrer`.

Note that the URL is configurable, so we can't hard-code it here. We need to use `openshiftHelpBase` from here:

https://github.com/openshift/console/blob/master/frontend/public/components/utils/documentation.tsx#L6-L7

Reply:
The issue I was running into is that hrefs added to the Action Menu all turned into unusable links like this: http://localhost:9000/k8s/ns/myproject/https://docs.okd.io/latest/

I looked at the dropdown component and couldn't figure out why it was happening.

Reply:
How were you building the link?

Reply:
<ActionsMenu
        actions={[
          {label: 'Documentation', href: openshiftHelpBase},
          {label: 'About', callback: this.open}]}
        noButton
        noCaret
        title={<i className="fa fa-question-circle-o co-masthead__help-icon" />} />

Reply:
Yeah, I think this will require changes to `ActionsMenu`. It's using `history.push`, which just adds to the path.